code/modules/mob/living/living_movement.dm

```diff
@@ -10,12 +11,13 @@
 		var/obj/projectile/P = mover
 		return !P.can_hit_target(src, P.permutated, src == P.original, TRUE)
 	if(mover.throwing)
-		return (!density || !(mobility_flags & MOBILITY_STAND) || (mover.throwing.thrower == src && !ismob(mover)))
+		return (!density || body_position == LYING_DOWN || (mover.throwing.thrower == src && !ismob(mover)))
 	if(buckled == mover)
 		return TRUE
 	if(ismob(mover) && (mover in buckled_mobs))
 		return TRUE
-	return (!mover.density || . || !(mobility_flags & MOBILITY_STAND))
+	return . || !mover.density || body_position == LYING_DOWN
```

Comment:
This works, but unwrapped ||'s are are not ordinarily used like this, so it's a bit ehhhh

Reply:
Why not? Subsequent `||` operations will be ignored if one truthy result is found first, and this makes it more explicit. It will return the first truthy value among those and ignore the rest.